Documentation/components/k8s/pcn-kubernetes.rst

```diff
@@ -44,7 +44,7 @@ Download and set up the following packages.
 
 **Follow the instruction below, after the pre-requisite:**
 
-1. Use this `PCN-K8S Vagrantfile <https://raw.githubusercontent.com/polycube-network/polycube/tests/vagrant/pcn-k8s/Vagrantfile>`_ for your setup.
+1. Use this `PCN-K8S Vagrantfile <https://raw.githubusercontent.com/polycube-network/polycube/master/tests/vagrant/pcn-k8s/Vagrantfile>`_ for your setup.
```

Comment:
As a note: This works fine for now, but once do the first release this will cause problems because the documentation of that release would point to this the vagrant file in master.
Using the `SCM_WEB` macro is better.